pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -350,7 +356,8 @@ class Timestamp(_Timestamp):
         second=None,
         microsecond=None,
         nanosecond=None,
-        tzinfo=None
+        tzinfo=None,
+        fold=None
```

Comment:
Should `fold` be keyword-only, like it is in `datetime.datetime`?

Reply:
Yes, we probably should. Done.